docs/user-guide/internationalization.md

```diff
@@ -248,6 +248,9 @@ The table below illustrates how the selection is made:
 | X | fr | jp | en | fr |
 | it | fr | jp | en | it |
 
+The translation service probes the browser culture first, for example `en-GB`.
+If the `en-GB.json` files does not exist, the service falls back to the language id: `en`.
```

Comment:
Should be "en-GB.json *file* does not exist" or "en-GB.json files do not exist".